app/src/main/java/umc/cozymate/ui/university_certification/UniversityCertificationFragment.kt

```
@@ -95,6 +95,10 @@ class UniversityCertificationFragment : Fragment() {
         }
     }
 
+    fun setFocusColor() {
+
+    }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)